tap_hubspot/client.py

```diff
@@ -79,6 +70,7 @@ def http_headers(self) -> dict:
             headers["User-Agent"] = self.config.get("user_agent")
         # If not using an authenticator, you may also provide inline auth headers:
         # headers["Private-Token"] = self.config.get("auth_token")
+        headers['Authorization'] = "Bearer" + self.config.get("api_key");
```

Comment:
I had code working with local edits in the .meltano folder. Now that I have made changes in GiHub and merged it in my own fork and use that with a custom extractor I notice that I forgot a space after "Bearer".